src/storageHandlers/localFolderHandler.ts

```diff
@@ -170,14 +176,14 @@ export class LocalFolderHandler extends EventEmitter implements StorageHandler {
 		}
 	}
 
-	getFile(name: string): Promise<File> {
+	getFile(name: string, previewFrame?: number): Promise<File> {
```

Comment:
Why do storage handlers need the knowledge of `previewFrame`? Wouldn't it be possible to just get if from a `TrackedMediaItem` and pass it to `generateNewFileWorkSteps` in workflow generators, and then as a property of `ScannerWorkStepInitial` in `ScannerWorkStep` constructor?